src/powershell/install-boxstarter.ps1

```diff
@@ -0,0 +1,43 @@
+# Install and configure Boxstarter
+# See: https://boxstarter.org/
+function Install-Boxstarter {
+    Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://boxstarter.org/bootstrapper.ps1'))
+    Get-Boxstarter -Force
```

Comment:
Is there a reason we cannot use the [Chocolatey installation directions](https://boxstarter.org/installboxstarter#installing-from-chocolatey)? I'm not a fan of pulling installation scripts from the web. If this method of installation is required please change the quotes for the URL to double quotes.

Reply:
@mcdonnnj resolved with [a526fd9](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/a526fd9b8491b6ba287a265ec7f67f1a294e58f5)

Reply:
@itsmostafa Would you add an issue around this? I would _strongly_ prefer that if possible it is installed with Chocolately, but understand if we cannot right now. I would like to track this as something to change in the future when the failure can be investigated (the error output is almost nonexistent so I have no clue why it's failing).

Reply:
@mcdonnnj I created an issue here: https://github.com/cisagov/windows-commando-vm-packer/issues/36

Please let me know if I covered everything